hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/mapreduce/MapReduceBackupMergeJob.java

```diff
@@ -310,8 +312,11 @@ protected void updateBackupManifest(String backupRoot, String mergedBackupId,
     List<String> backupsToDelete) throws IllegalArgumentException, IOException {
     BackupManifest manifest =
       HBackupFileSystem.getManifest(conf, new Path(backupRoot), mergedBackupId);
+    LOG.info("Removing ancestors from merged backup {} : {}", mergedBackupId, backupsToDelete);
     manifest.getBackupImage().removeAncestors(backupsToDelete);
     // save back
+    LOG.info("Creating new manifest file for merged backup {} at root {}", mergedBackupId,
+      backupRoot);
     manifest.store(conf);
```

Comment:
Not totally sure how/if this ever worked to be honest. We would deliberately create a manifest file for the merged backup, but then we'd fail to copy it out before purging the tmp directory so the final merged backup would be missing a manifest file.